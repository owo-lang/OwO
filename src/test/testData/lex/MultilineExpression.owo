a = b
  + c
e = k + d
f = do
  a +
    b
  c

g = do
   b +
 c
   d

inttest y = 1
  + 2

 + 3 + y
    + 4
