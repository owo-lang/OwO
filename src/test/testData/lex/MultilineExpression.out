IdentifierToken "a" (0 1 1) (1 1 2)
EqualToken (2 1 3) (3 1 4)
IdentifierToken "b" (4 1 5) (5 1 6)
OperatorToken "+" (8 2 3) (9 2 4)
IdentifierToken "c" (10 2 5) (11 2 6)
IdentifierToken "e" (12 3 1) (13 3 2)
EqualToken (14 3 3) (15 3 4)
IdentifierToken "k" (16 3 5) (17 3 6)
OperatorToken "+" (18 3 7) (19 3 8)
IdentifierToken "d" (20 3 9) (21 3 10)
IdentifierToken "f" (22 4 1) (23 4 2)
EqualToken (24 4 3) (25 4 4)
DoToken (26 4 5) (28 4 7)
BraceLToken (31 5 3) (31 5 3)
IdentifierToken "a" (31 5 3) (32 5 4)
OperatorToken "+" (33 5 5) (34 5 6)
IdentifierToken "b" (39 6 5) (40 6 6)
SemicolonToken (43 7 3) (43 7 3)
IdentifierToken "c" (43 7 3) (44 7 4)
BraceRToken (46 9 1) (46 9 1)
IdentifierToken "g" (46 9 1) (47 9 2)
EqualToken (48 9 3) (49 9 4)
DoToken (50 9 5) (52 9 7)
BraceLToken (56 10 4) (56 10 4)
IdentifierToken "b" (56 10 4) (57 10 5)
OperatorToken "+" (58 10 6) (59 10 7)
BraceRToken (61 11 2) (61 11 2)
IdentifierToken "c" (61 11 2) (62 11 3)
IdentifierToken "d" (66 12 4) (67 12 5)
IdentifierToken "inttest" (69 14 1) (76 14 8)
IdentifierToken "y" (77 14 9) (78 14 10)
EqualToken (79 14 11) (80 14 12)
IntegerToken 1 (81 14 13) (82 14 14)
OperatorToken "+" (85 15 3) (86 15 4)
IntegerToken 2 (87 15 5) (88 15 6)
OperatorToken "+" (91 17 2) (92 17 3)
IntegerToken 3 (93 17 4) (94 17 5)
OperatorToken "+" (95 17 6) (96 17 7)
IdentifierToken "y" (97 17 8) (98 17 9)
OperatorToken "+" (103 18 5) (104 18 6)
IntegerToken 4 (105 18 7) (106 18 8)
